lib-ltc/include/ltcdisplayrgbpanel.h

```diff
@@ -32,6 +32,10 @@
 
 #include "rgbpanel.h"
 
+#define RGBPANEL_TEXT_MAX_COLS 32 
+#define RGBPANEL_TEXT_MAX_LINES 4
+
```

Comment:
#define is C, in modern C++ constexpr is used in namespaces. 

There is already an enum LTCDISPLAY_MAX_MESSAGE_SIZE , that should also get a namespace